pkg/features/embedded/boot-assessment/usr/sbin/elemental-boot-assessment

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
We could probably take this even further and add a `elemental health-check` command. Then we add some test-coverage to this code :+1:

Reply:
Yes I was also thinking about it. But first wanted to actually verify the approach. I have also been thinking on including the boot-assessment logic in grub.cfg directly and just use a variable to enable it or not.

Reply:
For I think I'll leave it as this simple script. Let's reconsider this once we have the boot assessment in place and functional all the way up to Micro images.